ios/FluentUI/Bottom Commanding/BottomCommandingController.swift

```diff
@@ -322,6 +327,7 @@ open class BottomCommandingController: UIViewController {
             popoverContentViewController.view.addSubview(tableView)
             popoverContentViewController.modalPresentationStyle = .popover
             popoverContentViewController.popoverPresentationController?.sourceView = moreButtonView
+            popoverContentViewController.preferredContentSize = CGSize(width: 0, height: fittingTableViewHeight)
```

Comment:
why zero width?

Reply:
I'll double check if this is documented, but providing zero width results in the system setting the width automatically, just like if I didn't provide a preferredContentSize at all.

Reply:
Turns out I can directly just set the height, so I'll do that.